### ISSUE TITLE
Daily Docket | Move back to hearing days up to top

### ISSUE THREAD

**Issue by annienguyenUSDS** [department-of-veterans-affairs/caseflow#4771]
To be consistent with navigation in Queue, move "< Back to Your Hearing Days" link from bottom of page to top left in gray space above hearing worksheet

@annienguyenUSDS 
option 1:
![back](https://user-images.githubusercontent.com/4975959/38470689-9c8d0e50-3b34-11e8-90a9-88b315691643.png)

**Comment by annienguyenUSDS** [comment]
@lakohl this would be good to align navigation and make sure where they landed.

**Comment by annienguyenUSDS** [comment]
Messaged Queue, Phillip is working on Queue navigation and will get back to us.

**Comment by lakohl** [comment]
@Chingujo just checking if Queue has created any new navigation patterns yet?

**Comment by Chingujo** [comment]
We discussed a broader navigation conversation across applications yesterday =D! We are doing some explorations on how best to navigate between apps like reader and hearings for this sprint. 

We were planning to present to the broader design team once we finished (@mkhandekar @sneha-pai) for awareness. 

At a more granular level we made the following in app decisions for embedded flows (i.e. for the attorney checkouts):

- No bread crumbs as they were creating complex navigations and logic. 
- A cancel, back, and continue buttons as seen below

![footer_buttons](https://user-images.githubusercontent.com/8533004/39596020-224d8976-4edf-11e8-8100-24b981be9298.gif)
